hw/ip/adc_ctrl/data/adc_ctrl.hjson

```diff
@@ -11,11 +11,12 @@
   To enable usage while the device is sleeping, it runs on a slow always-on clock.
   In addition, it has a low power periodic scan mode for monitoring ADC channels.
   '''
+  cip_id:             "1",
```

Comment:
I think it would be worth to include greppable reference here to find the file where all the CIP IDs are defined. Otherwise it can be tricky to find those. It doesn't need to be the full path to the file because this might change over time and we might not want to update all IP hjsons when we do so. But what could work is to add a comment here saying something like "Unique comportable IP identifier defined under KNOWN_CIP_IDS" and then people can search for KNOWN_CIP_IDS.

Reply:
done!